modules/integration/src/test/java/org/apache/fluo/integration/impl/WorkerIT.java

```diff
@@ -42,7 +44,9 @@
  * index of node degree.
  */
 public class WorkerIT extends ITBaseMini {
-
+  @Rule
+  public Timeout globalTimeout = Timeout.seconds(120);
```

Comment:
If using the static method, this could be modified to the following

```java
  @Rule
  public Timeout globalTimeout = Timeout.seconds( 2 * getTestTimeout());
```

Reply:
Ah, that is a nice way to do it.